index.rst

```diff
@@ -13,6 +13,7 @@ Contents:
    general/technologies
    general/glossary
    general/code-of-conduct
+   developer/introduction
```

Comment:
rename the page to reflect what I wrote in the previous comments and move it under `contributing` please,

Reply:
I don't know if the name I put now it good. Lemme know yourself.